### PR TITLE
Add documentation for stack commands

### PR DESCRIPTION
**Pull request by vdemeester**
We already document _experimental_ commands like `plugin` subcommands, adding some documentation for `stack` subcommands and for the `deploy` command.

Also removes the `-f` flags of bundle to follow the single-letter flags evaluation.
- [ ] Add more example on `tasks` subcommand (like what we do on `service tasks`).
- [ ] What to do about the short flag for `--bundle`.

/cc @thaJeztah @dnephin @aanand @tiborvass @icecrime 

Signed-off-by: Vincent Demeester vincent@sbr.pm

### CODE REVIEW

File: api/client/stack/opts.go [hunk 1]
```diff
@@ -12,9 +12,9 @@ import (
 )
 
 func addBundlefileFlag(opt *string, flags *pflag.FlagSet) {
-	flags.StringVarP(
+	flags.StringVar(
 		opt,
-		"bundle", "f", "",
+		"bundle", "",
```
Comment (thaJeztah) [inline api/client/stack/opts.go]: Yeah, still wondering if we should change this to `--file`

@dnephin @aanand wdyt?

File: docs/reference/commandline/deploy.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Loading bundle from vossibility-stack.dab
+Creating service vossibility-stack_elasticsearch
+Creating service vossibility-stack_kibana
+Creating service vossibility-stack_logstash
+Creating service vossibility-stack_lookupd
+Creating service vossibility-stack_nsqd
+Creating service vossibility-stack_vossibility-collector
+```
+
+We can verify that services were correctly created:
```
Comment (thaJeztah) [inline docs/reference/commandline/deploy.md]: s/We can/You can/

Also misses a "the";

```Markdown
You can verify that the services were correctly created:
```

File: docs/reference/commandline/stack_deploy.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Loading bundle from vossibility-stack.dab
+Creating service vossibility-stack_elasticsearch
+Creating service vossibility-stack_kibana
+Creating service vossibility-stack_logstash
+Creating service vossibility-stack_lookupd
+Creating service vossibility-stack_nsqd
+Creating service vossibility-stack_vossibility-collector
+```
+
+We can verify that services were correctly created:
```
Comment (thaJeztah) [inline docs/reference/commandline/stack_deploy.md]: Same here :)

**Comment by thaJeztah** [comment]
some nits/questions, but otherwise LGTM

File: api/client/stack/opts.go (hunk 1 above)
Comment (dnephin) [inline api/client/stack/opts.go]: I'm fine with either

Comment (thaJeztah) [inline api/client/stack/opts.go]: It feels more consistent with `docker build --file`, and `docker-compose --file`

Comment (thaJeztah) [inline api/client/stack/opts.go]: But better do that in a separate PR 👍

**Comment by icecrime** [comment]
LGTM

**Comment by cpuguy83** [comment]
LGTM
